save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TestExecutionRepository.kt

```diff
@@ -44,6 +50,17 @@ interface TestExecutionRepository : BaseEntityRepository<TestExecution>, JpaSpec
     @Suppress("MISSING_KDOC_ON_FUNCTION", "MISSING_KDOC_CLASS_ELEMENTS")
     fun findByExecutionIdAndStatus(executionId: Long, status: TestResultStatus, pageable: Pageable): List<TestExecution>
 
+    @Suppress("MISSING_KDOC_ON_FUNCTION", "MISSING_KDOC_CLASS_ELEMENTS")
+    fun findByExecutionIdAndTestTestSuiteName(executionId: Long, name: String, pageable: Pageable): List<TestExecution>
+
+    @Suppress("MISSING_KDOC_ON_FUNCTION", "MISSING_KDOC_CLASS_ELEMENTS")
+    fun findByExecutionIdAndStatusAndTestTestSuiteName(
```

Comment:
What do you think about using something like [this](https://www.baeldung.com/spring-data-jpa-null-parameters#Ignoring) to avoid creation of a lot of similar methods?

Reply:
Yes. We can use this, but in this case it will turn out to be a very large request. TestExecution don't have a field for the suite name. We will have to make two join with the Test and TestSuite tables.

Reply:
Yeah, I see. Let's leave it as is.